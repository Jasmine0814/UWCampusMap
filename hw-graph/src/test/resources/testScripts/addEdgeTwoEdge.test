# test that create a graph and add two nodes, two edges can be added between two node in a graph

# create a graph
created graph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Add an edge between the two nodes
AddEdge graph1 n1 n2 e1
AddEdge graph1 n2 n1 e2

# Print the nodes in the graph and the edges from n1 to n2 and from n2 to n1
ListNodes graph1 n1
ListChildren graph1 n1
ListChildren graph1 n2