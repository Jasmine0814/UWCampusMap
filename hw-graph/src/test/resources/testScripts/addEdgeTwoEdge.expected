# test that create a graph and add two nodes, two edges can be added between two node in a graph

# Create a Graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Add two edges between the two nodes
added edge e1 from n1 to n2 in graph1
added edge e2 from n2 to n1 in graph1

# Print the nodes and their child
graph1 contains: n1 n2
the children of n1 in graph1 are: n2(e1)
the children of n2 in graph1 are: n1(e2)